src/OSPSuite.Presentation/Presenters/Charts/CurveMultiItemEditorPresenter.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using OSPSuite.Core.Chart;
+using OSPSuite.Core.Domain;
+using OSPSuite.Presentation.Views.Charts;
+using OSPSuite.Utility;
+
+namespace OSPSuite.Presentation.Presenters.Charts
+{
+   public interface ICurveMultiItemEditorPresenter : IDisposablePresenter, IPresenter<ICurveMultiItemEditorView>
+   {
+      SelectedCurveValues GetSelectedValues();
+      IEnumerable<bool?> AllBooleanOptions { get; }
+      IEnumerable<LineStyles?> AllLineStyles { get; }
+      IEnumerable<Symbols?> AllSymbols { get; }
+      void SetColorChangedFlag();
+   }
+
+   public class SelectedCurveValues
+   {
+      public Color Color { get; set; }
+      public LineStyles? Style { get; set; }
+      public Symbols? Symbol { get; set; }
+      public bool? Visible { get; set; }
+      public bool? VisibleInLegend { get; set; }
+      public bool IsColorSet { get; set; } = false;
+   }
+
+   public class CurveMultiItemEditorPresenter : AbstractDisposablePresenter<ICurveMultiItemEditorView, ICurveMultiItemEditorPresenter>,
+      ICurveMultiItemEditorPresenter
+   {
+      private SelectedCurveValues _selectedCurveValues;
```

Comment:
Run resharper on this file. It's clearly not formatter properly. Let's run resharper on all files before we submit a PR

Reply:
good point, I do go through the changes, but just by looking at the diff it is easy to lose something.